beacon_node/execution_layer/src/engine_api.rs

```diff
@@ -277,3 +303,20 @@ pub struct ProposeBlindedBlockResponse {
     pub latest_valid_hash: Option<Hash256>,
     pub validation_error: Option<String>,
 }
+
+// This name is work in progress, it could
+// change when this method is actually proposed
+// but I'm writing this as it has been described
+#[derive(Clone, Copy)]
+pub struct SupportedApis {
+    // I wonder if it would be better to use a bitmask here..
+    // It would be 8x more space efficient but I do remember
+    // Mac saying something about the performance being bad..
```

Comment:
I think it's fine as-is because we only store a couple of these, and the compiler will pack `SupportedApis` so that it fits inside a `usize` anyway (7 bools < 1 word)

Reply:
Awesome! Didn't know the compiler would do that.